# Constants from controlling the car via node

# cmd_vel
float64 cv_speed

float64 cv_turn_zero
float64 cv_turn_speed

# drive_pwm
int16 vp_speed

int16 vp_turn_zero
int16 vp_turn_speed

# VESC
float64 vesc_speed
